srcpkgs/json-table/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'json-table'
+pkgname=json-table
+version=2.0.0
+revision=1
+build_style=gnu-makefile
+short_desc="Transform nested JSON data into tabular data in the shell"
+maintainer="akhil <akhil.j@outlook.in>"
+license="EPL-1.0"
+homepage="https://github.com/micha/json-table"
+changelog="https://github.com/micha/json-table/releases"
+distfiles="https://github.com/micha/json-table/archive/refs/tags/${version}.tar.gz"
+checksum=6e74d0b5b82d2f04c069302d0c477aa7993012602da5dffb600e472049388775
+
+do_install() {
+	vbin jt
+	vman jt.1
+}
```

Comment:
It's unfortunate that their `Makefile` doesn't respect [`$DESTDIR`](https://www.gnu.org/software/make/manual/html_node/DESTDIR.html). Well written `Makefile`s should support that. Might be worth telling the developers about it.

Reply:
yes. informed developer. https://github.com/micha/json-table/issues/14

Reply:
If it would utilize `$DESTDIR`, than you wouldn't have to specify `do_install()` at all because it would be handled automatically by the `gnu-makefile` build style. Specifying a non-default install directory via `$DESTDIR` is essential for package managers because they install the important files to a "fake" Linux system which contains just the package and then process this "fake" system to make it usable with XBPS.

The page I linked (https://www.gnu.org/software/make/manual/html_node/DESTDIR.html) and [the Manual](https://github.com/void-linux/void-packages/blob/master/Manual.md) contains more information.